tracer/test/Datadog.Trace.Security.IntegrationTests/ApiSecurity/AspNetCore5Endpoints.cs

```diff
@@ -33,5 +34,14 @@ public class AspNetCore5EndpointsEnabledLimited(AspNetCoreTestFixture fixture, I
 
     public class AspNetCore5EndpointsDisabled(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
         : AspNetCoreEndpoints(fixture, outputHelper, sampleName: "AspNetCore5", false);
+
+    public class AspNetCore5EndpointsApmTracingDisabled : AspNetCoreEndpoints
+    {
+        public AspNetCore5EndpointsApmTracingDisabled(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
+            : base(fixture, outputHelper, sampleName: "AspNetCore5", true)
+        {
+            SetEnvironmentVariable(ConfigurationKeys.ApmTracingEnabled, "0");
+        }
+    }
```

Comment:
Could this test be also added to [AspNetCoreBareEndpoints.cs](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/Datadog.Trace.Security.IntegrationTests/ApiSecurity/AspNetCoreBareEndpoints.cs) (or this is too redundant?)